definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-prod.json

```diff
@@ -37,21 +37,7 @@
   {
     "LiveFrom": "20/09/2019",
     "CaseTypeID": "DIVORCE",
-    "CaseEventID": "EnterAOSOffline_Respondent_fromAosAwaiting",
-    "UserRole": "caseworker-divorce-courtadmin_beta",
-    "CRUD": "R"
-  },
-  {
-    "LiveFrom": "20/09/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseEventID": "EnterAOSOffline_Respondent_fromAosStarted",
-    "UserRole": "caseworker-divorce-courtadmin_beta",
-    "CRUD": "R"
-  },
-  {
-    "LiveFrom": "20/09/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseEventID": "EnterAOSOffline_Respondent_fromAosOverdue",
+    "CaseEventID": "EnterAOSOffline_Respondent",
```

Comment:
In theory, you could have this in the definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json file. But I'll leave it up to you.

Reply:
I have it there except for the caseworker-divorce-courtadmin_beta role because the value of CRU needs to be different for AAT and Prod.
If I put it also in the AuthorisationCaseEvent.json I think it gives a 'duplicate issue' :)

Reply:
got you.